lighthouse-core/report/templates/report.html

```diff
@@ -63,73 +70,97 @@ <h1 class="menu__header-title">Lighthouse</h1>
         </div>
       </div>
 
-      <div class="report-body__url">Results for <a href="{{ url }}" target="_blank">{{ url }}</a></div>
-
+      <div class="aggregations">
       {{#each aggregations}}
-      <div class="js-breakdown report-body__breakdown">
-        <h1 class="report-section__title" id="{{nameToLink this.name}}">
-          <span class="report-section__title-main">{{ this.name }}</span>
-          {{#if this.scored}}
-          <span class="report-section__title-score">
-            <span class="report-section__title-score-total report-section__title-score-total--{{ getTotalScoreRating this }}">{{ getTotalScore this }}</span>
-            <span class="report-section__title-score-max"> / 100</span>
+      <section class="js-breakdown section1">
+        <header class="header1" id="{{nameToLink this.name}}">
+          <h1>{{ this.name }}</h1>
+        </header>
+        <p class="section1__desc">{{ this.description }}</p>
+        {{#if this.scored}}
+        <div class="section-result">
+          <span class="section-result__score --{{ getTotalScoreRating this }}">
+            <span class="section-result__points">{{ getTotalScore this }}</span>
+            <span class="section-result__divider">/</span>
+            <span class="section-result__total">100</span>
           </span>
-          {{/if}}
-        </h1>
-        <p class="report-section__description">{{ this.description }}</p>
+        </div>
+        {{/if}}
 
         <div class="js-report-by-user-feature">
           {{#each this.score as |aggregation|}}
-            <section class="report-body__breakdown-item">
+            <section class="section2">
```

Comment:
That's tough. In the User Feature view, these represent aggregations. In Technology view, they represent categories. Both are styled identically. Is there a generic term that describes both of these? I'm following the HTML convention of H1 for level-1 headings, H2 for level-2 headings, etc. but am open to alternatives.